pycbc/conversions.py

```diff
@@ -427,13 +427,15 @@ def primary_xi(mass1, mass2, spin1x, spin1y, spin2x, spin2y):
     """Returns the effective precession spin argument for the larger mass.
     """
     spinx = primary_spin(mass1, mass2, spin1x, spin2x)
-    spiny = secondary_spin(mass1, mass2, spin1y, spin2y)
+    spiny = primary_spin(mass1, mass2, spin1y, spin2y)
     return chi_perp_from_spinx_spiny(spinx, spiny)
 
 def secondary_xi(mass1, mass2, spin1x, spin1y, spin2x, spin2y):
     """Returns the effective precession spin argument for the smaller mass.
     """
-    return xi2_from_mass1_mass2_spin2x_spin2y(mass1, mass2, spin2x, spin2y)
+    spinx = secondary_spin(mass1, mass2, spin1x, spin2x)
+    spiny = secondary_spin(mass1, mass2, spin1y, spin2y)
+    return xi2_from_mass1_mass2_spin2x_spin2y(mass1, mass2, spinx, spiny)
```

Comment:
Can we just have `xi2_from_mass1_mass2_spinx_spiny` instead of `xi2_from_mass1_mass2_spin2x_spin2y`? The name of the function suggests that xi2 depends on the x and y components of the secondary component's spin, whereas it actually depends on the x and y components of the primary's as well as the secondary component's spin (going into spinx and spiny). If so, can we rename spin2x and spin2y to spinx and spiny respectively in line 446? Same question for `xi1_from_spin1x_spin1y` ... can we rename that to xi1_from_spinx_spiny and make its arguments spinx and spiny instead of spin1x and spin1y?